webextension/background.js

```diff
@@ -6,14 +6,136 @@
 
 /* globals browser */
 
-const gURLThankYou = "https://mozilla.github.io/webcompat-blipz-experiment/thanks.html";
+let gCurrentlyPromptingTabId;
+
+const Config = (function() {
+  class Config {
+    constructor() {
+      this._neverShowAgain = false;
+      this._skipPrivateBrowsingTabs = true;
+      this._lastPromptTime = 0;
+      this._domainsToCheck = {
```

Comment:
I wonder if we should put the list of domains in a pref, then construct this object from that. Probably for the experiment it doesn't matter much...

Reply:
Yes, this can just be a placeholder for now. I was thinking I would have the addon compare this in-code list to the about:config pref, to detect changes to the list and be able to add/remove URLs as the addon is changed.